app/helpers/passengers_helper.rb

```diff
@@ -11,13 +13,30 @@ def doctors_note_row_class(note)
     end
   end
 
-  def passengers_table_row_class(passenger)
-    return if passenger.permanent?
-    return 'no_note' if passenger.doctors_note.blank?
+  def longer_ride(device)
```

Comment:
Probably good to stick with the convention that these class helpers end with `_class`